thirdparty/xmlstream/xmlstream.cpp

```diff
@@ -2116,7 +2116,8 @@ QString XmlStreamReader::readElementText(ReadElementTextBehaviour behaviour)
                     result += readElementText(behaviour);
                     break;
                 }
-                [[gnu::fallthrough]];
+                // fall through
+                //(for ErrorOnUnexpectedElement)
```

Comment:
I think to have read that the fall thru comment needs to reight before the next case or default statement?